src/game_inventory.cpp

```diff
@@ -78,6 +78,47 @@ using item_location_filter = std::function<bool ( const item_location & )>;
 namespace
 {
 
+/**
+ * Return digits in base 10 ignoring minus sign.
+ *
+ * It exists as faster alternative to *_justify for justifiing integers.
+ * source: https://stackoverflow.com/a/4143288/5057078
+ */
+int n_digits( int i )
+{
+    if( i < 0 ) {
+        i = -i;
+    }
+    if( i <         10 ) {
+        return 1;
+    }
+    if( i <        100 ) {
+        return 2;
+    }
+    if( i <       1000 ) {
+        return 3;
+    }
+    if( i <      10000 ) {
+        return 4;
+    }
+    if( i <     100000 ) {
+        return 5;
+    }
+    if( i <    1000000 ) {
+        return 6;
+    }
+    if( i <   10000000 ) {
+        return 7;
+    }
+    if( i <  100000000 ) {
+        return 8;
+    }
+    if( i < 1000000000 ) {
+        return 9;
+    }
+    return 10;
```

Comment:
Try returning log10( i ) + 1 (which as I understand it will get rounded down) instead of just assuming 10? (Not needed for the below use, but for future uses.)

Reply:
I guess your version will not be reached that often, so performance-wise it is negligible and therefore ok to do.